rust/noosphere-ns/src/lib.rs

```diff
@@ -14,13 +14,15 @@ mod records;
 pub mod utils;
 mod validator;
 
+pub mod helpers;
```

Comment:
There's `mod utils` which serves a similar purpose, though `helpers` specifically is for tests -- maybe can consolidate these into a single module, or have helper module (or functions) gated on `cfg(test)`. I don't think `NsData` should be exposed by default.

Reply:
I tried gating this on `cfg(test)` and then on a feature flag, but there is no satisfying way that I could find where this stuff is _only_ exposed to tests in both integration tests and other crates. Open to suggestions.